wolfcrypt/src/random.c

```diff
@@ -1102,20 +1102,20 @@ int wc_RNG_HealthTest_ex(int reseed, const byte* nonce, word32 nonceSz,
 }
 
 
-const byte seedA[] = {
+const FLASH_QUALIFIER byte _seedA[] = {
```

Comment:
Avoid using _ prefix.  That is reserved for compilers and internal tools.

Reply:
Ok, I can rename these variables, and resubmit